build.sh

```diff
@@ -220,6 +220,30 @@ case "$COMMAND" in
 esac
 export CONFIGURATION
 
+xcode_for_swift() {
+    local xcodes dev_dir
+    xcodes=()
+    dev_dir="Contents/Developer"
+    for dir in $(mdfind "kMDItemCFBundleIdentifier == 'com.apple.dt.Xcode'" 2>/dev/null); do
+        [[ -d "$dir" && -n "$(ls -A "$dir/$dev_dir")" ]] && xcodes+=("$dir/$dev_dir")
+    done
+
+    for dir in "${xcodes[@]}"; do
+        version="$("$dir/usr/bin/xcrun" swift --version 2>/dev/null | sed -ne 's/^Apple Swift version \([^\b ]*\).*/\1/p')"
+        if [[ "$version" = "$1" ]]; then
+            echo "$dir"
+            break;
+        fi
+    done
+}
+
+: ${REALM_SWIFT_VERSION:=1.2}
+: ${DEVELOPER_DIR:="$(xcode_for_swift "$REALM_SWIFT_VERSION")"}
+export DEVELOPER_DIR
+if [[ "$REALM_SWIFT_VERSION" = "1.2" ]]; then
+   REALM_SWIFT_VERSION=
```

Comment:
Deos `build.sh` use two space or four space indentation? This change contains a mix.